pkg/summary/gvk.go

```diff
@@ -0,0 +1,100 @@
+package summary
+
+import (
+	"encoding/json"
+	"errors"
+	"fmt"
+	"strings"
+
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/apimachinery/pkg/util/sets"
+)
+
+const (
+	conditionStatusSep = "/"
+	gvkFormat          = "%s" + conditionStatusSep + "%s" + conditionStatusSep + "%s"
```

Comment:
This isn't separating the status, it's separating the GVK parts.

Also, we shouldn't invent a new format for stringifying GVKs. GVKs already have a `String()` method that formats them in a preferred format that we should reuse: `helm.cattle.io/v1, Kind=HelmChart`

See the helpers at https://github.com/k3s-io/k3s/blob/c2efae3e1cf4a3e761ec297edd542504d7166b46/pkg/deploy/controller.go#L507-L524